src/pages/Onboard/NameTeam.vue

```diff
@@ -165,6 +257,29 @@ export default {
           <v-col v-if="revealNote" key="name" cols="12" class="pb-0">
             <div class="display-1 text-center">
               Let's start by creating your team
+              <v-menu
+                :close-on-content-click="false"
+                offset-y
+                transition="slide-y-transition"
+              >
+                <template #activator="{ on }">
+                  <v-icon class="white--text" v-on="on"
+                    >fa-question-circle</v-icon
+                  >
+                </template>
+                <v-card tile class="pa-2 mt-1" max-width="320">
+                  <div class="body-2">
```

Comment:
```suggestion
                  <div class="body-1">
```
This is a little small at the moment, can we increase the text size a little and add some more padding?

Reply:
should be better now!